dev/flex/main.tf

```diff
@@ -7,6 +7,11 @@ terraform {
       version = "~> 5.2"
     }
 
+    helm = {
+      source  = "hashicorp/helm"
+      version = "~> 2.9"
+    }
+
     http = {
```

Comment:
I think this is no longer necessary now that multus isn't installed here?

Think this probably goes for a few modules.